src/applications/vaos/appointment-list/components/AppointmentsPageV2/RequestListItem.jsx

```diff
@@ -21,25 +25,36 @@ export default function RequestListItem({ appointment, facility }) {
             Canceled
           </span>
         )}
-        <h4 className="vads-u-font-size--h4 vads-u-margin-x--0 vads-u-margin-y--0">
+        <h3 className="vads-u-font-size--h4 vads-u-margin-x--0 vads-u-margin-y--0">
```

Comment:
I think @vbahinwillit made this same fix in another PR, you two can decide which PR should make the change.

Reply:
Hi @vbahinwillit, I made the same heading change you did, along with few other things, for the file "RequestListItem.jsx". This file will merge with master today. Sorry I didn't put one and one together when you where giving your standup.